frontend/src/metabase/ui/components/inputs/SegmentedControl/SegmentedControl.styled.tsx

```diff
@@ -51,10 +51,12 @@ export const getSegmentedControlOverrides =
             backgroundColor: theme.fn.themeColor("bg-medium"),
           },
           controlActive: {
-            backgroundColor: shouldAnimate ? theme.white : undefined,
+            backgroundColor: shouldAnimate
+              ? theme.fn.themeColor("bg-white")
+              : undefined,
           },
           indicator: {
-            backgroundColor: theme.white,
+            backgroundColor: theme.fn.themeColor("bg-white"),
           },
```

Comment:
Allows the segmented control to be themed:

![CleanShot 2567-11-25 at 18 40 32@2x](https://github.com/user-attachments/assets/91adfa48-a099-4556-bb85-4fe775556234)